telosnetwork/telos-wallet
# #157 | Home page UI

# Fixes #157 

## Description
This PR adds the UI for the landing page. note that the particle effect video has been omitted from the bottom of the page because the video has a black background, we decided in the chat to leave it out

## Test scenarios

- go to https://deploy-preview-193--wallet-staging.netlify.app/
    - it should match the designs and be keyboard accessible
- click Native and log in e.g. cleos
    - native login should work as before
- navigate back after logging in
- select Native tab
    - you should see buttons to go to Balances or log out
    - both buttons should work
    - your native username should be displayed 

## Checklist:
